packages/web-components/src/components/ic-accordion/ic-accordion.css

```diff
@@ -106,6 +106,8 @@ button:disabled {
   display: flex;
   flex-direction: column;
   visibility: var(--ic-expanded-content-visibility, hidden);
+  --ic-typography-color: var(--ic-accordion-body-text);
```

Comment:
same: i think needs an empty line above it, or i think can move --vars to the top of the block

Reply:
I wasn't getting lint errors / warnings at all for some reason - but have moved this (and other `--ic-typography`s) to the top of their blocks, so hopefully that is ok!